src/core/document.js

```diff
@@ -1815,6 +1819,15 @@ class PDFDocument {
       }
     }
 
+    if (
+      parentRef &&
+      field.get("Subtype") === Name.get("Widget") &&
```

Comment:
Given that `Name` objects are cached (similar to `Ref` and `Cmd`) this isn't generally safe, however we have a utility function for such cases.
```suggestion
      isName(field.get("Subtype"), "Widget") &&
```